smart-contracts/wasm-chain-integration/src/v1/ffi.rs

```diff
@@ -228,6 +228,9 @@ unsafe extern "C" fn call_receive_v1(
     output_return_value: *mut *mut ReturnValue,
     output_config: *mut *mut ReceiveInterruptedStateV1,
     output_len: *mut size_t,
+    // This is set to 1 if and only if the stage has changed before the end of execution,
+    // in case execution terminated normally, without a runtime exception.
```

Comment:
```suggestion
    // This is set to 1 if and only if the state has changed before the end of an execution that terminated normally.
    // Normal in the sense that there was no runtime exception.
```

~stage~ state, and I tried to make it more clear.

Reply:
Revised